commitlint.config.js

```diff
@@ -0,0 +1,38 @@
+
+module.exports = {
+  parserPreset: 'conventional-changelog-conventionalcommits',
+  rules: {
+    'body-leading-blank': [1, 'always'],
```

Comment:
@asadikhlas what is this *1*? It seems there is a number in each rule, what does it mean? Is it levels of message? 1 for error and 2 for warning?